com.microsoft.mrtk.core/Subsystems/Speech/DictationEventArgs.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.MixedReality.Toolkit.Subsystems
+{
+    /// <summary>
+    /// Event data associated with the result of dictation.
+    /// </summary>
+    public class DictationResultEventArgs
```

Comment:
are these event args reused or one-time-use? if the latter, could they be rewritten as [readonly structs](https://learn.microsoft.com/en-us/dotnet/csharp/write-safe-efficient-code#declare-immutable-structs-as-readonly) to more clearly express the design intent?